tests/test_infosets.py

```diff
@@ -4,6 +4,14 @@
 from . import games
 
 
+def _create_game() -> gbt.Game:
```

Comment:
This function can use a more specific name.

However in this case if you're building the game, you're actually already testing `append_move` and `append_infoset` so probably these calls are best to be in the test function directly.